pkg/config/autodiscover/deployment_info_test.go

```diff
@@ -117,7 +117,7 @@ func TestGetTargetDeploymentByNamespace(t *testing.T) {
 			Prefix: "prefix1",
 			Name:   "name1",
 			Value:  "value1",
-		})
+		}, "deploymnet")
```

Comment:
There's a typo here, it should be "deployment"

Reply:
you can use deployment_info.Deployment constant in here

Reply:
This is not fixed yet